test/test_modules.py

```diff
@@ -514,6 +514,109 @@ def _to_device(obj):
                 self.assertEqual(cpu_grad_kwarg_input, gpu_grad_kwarg_input)
 
 
+    @modules(module_db)
+    def test_memory_format(self, device, dtype, module_info):
+        module_cls = module_info.module_cls
+        module_inputs = module_info.module_inputs_func(module_info, device=device, dtype=dtype,
+                                                       requires_grad=False)
+        is_conv = 'Conv' in module_cls.__name__
+        cudnn_greater_7_6 = False
+        cudnn_greater_8 = False
+        if ('cuda' in device) and torch.backends.cudnn.is_available():
+            cudnn_greater_7_6 = torch.backends.cudnn.version() >= 7603
+            cudnn_greater_8 = torch.backends.cudnn.version() >= 8005
```

Comment:
I think it would be better to have this logic specified in the conv ModuleInfo entries somehow - do you think this is possible somehow?